packages/components/src/mobile/picker/index.android.js

```diff
@@ -35,30 +46,43 @@ export default class Picker extends Component {
 	}
 
 	onCellPress( value ) {
-		this.props.onChange( value );
+		const { onChange } = this.props;
+		onChange( value );
 		this.onClose();
 	}
 
 	render() {
+		const { options, leftAlign, hideCancelButton } = this.props;
+		const { isVisible } = this.state;
+
 		return (
 			<BottomSheet
-				isVisible={ this.state.isVisible }
+				isVisible={ isVisible }
 				onClose={ this.onClose }
 				style={ { paddingBottom: 20 } }
 				hideHeader
 			>
 				<View>
-					{ this.props.options.map( ( option, index ) => (
-						<BottomSheet.Cell
-							icon={ option.icon }
-							key={ index }
-							leftAlign={ this.props.leftAlign }
-							label={ option.label }
-							separatorType={ 'none' }
-							onPress={ () => this.onCellPress( option.value ) }
-						/>
+					{ options.map( ( option, index ) => (
+						<>
+							{ options.length > 1 && option.separated && (
```

Comment:
Have you tried to use `separatorType` from the `BottomSheet.Cell`?

Reply:
In that case we have different requirements for that separator (it has additional vertical padding), so I've decided to create a custom one.